### PR TITLE
Added Gitpod to Full IDE

### PR DESCRIPTION
**Pull request by svenefftinge**
Fixes styfle/awesome-online-ide#60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,12 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
...
 - [Coder](https://coder.com) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run most languages on a full Linux VM complete with a terminal, go-to-definition, linting, live collaboration, and auto-scaling.
+- [Gitpod](https://gitpod.io) <img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" /> <img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" /> <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run any language on a full Linux VM complete with terminals, GitHub and Git integration, content assist, go-to-definition, linting, live collaboration, custom Docker workspaces, and integrated code review support.
```
Comment (styfle) [inline README.md]: The GitHub image here should link to the github repo (see other examples above).
The Docker image here should link to the docker hub repo (or self-host instructions).

**Comment by styfle** [comment]
@svenefftinge Thanks for the PR! I left a couple comments.

**Comment by svenefftinge** [comment]
@styfle Thanks. I've updated the PR

**Comment by svenefftinge** [comment]
done

**Comment by styfle** [comment]
Perfect, thanks 👌

**Pull request merged**